packages/ethereum-js-wallet-provider/lib/EthereumJsWalletProvider.js

```diff
@@ -105,6 +106,21 @@ export default class EthereumJsWalletProvider extends Provider {
     return normalizeTransactionObject(formatEthResponse(txData))
   }
 
+  async sendSweepTransaction (externalAddress, _gasPrice) {
+    const addresses = await this.getAddresses()
+
+    const balance = await this.getMethod('getBalance')(addresses)
+
+    const [ gasPrice ] = await Promise.all([
+      _gasPrice ? Promise.resolve(_gasPrice) : this.getMethod('getGasPrice')()
```

Comment:
I think this comes back as wei? Later on in `sendTransaction` we pass this as the fee parameter (`gasPrice`) and on that level it expects it in gwei.

Reply:
So `getGasPrice` function returns the value in `gwei`

So I'm pretty sure gasPrice in `gwei` is being passed around (otherwise there would be issues here: https://github.com/liquality/chainabstractionlayer/blob/dev/packages/ethereum-js-wallet-provider/lib/EthereumJsWalletProvider.js#L96 )